oaknational/Oak-Web-Application
# feat: lazily load mux player to save on blocking js

## Description

- use `mux-player-react/lazy`

## Issue(s)

Fixes #1401 

## How to test

1. Go to {c[loud link}](https://deploy-preview-1400--oak-web-application.netlify.thenational.academy/webinars/how-oak-can-reduce-your-workload-and-support-in-your-training-year-and)
2. open dev tools, filter by js
3. Fill out form so that video is available
4. You should see a 108kb package has loaded ... that was mux, and previously I believe it was being loaded on every page

## Screenshots

Placeholder is now black apparently

![image](https://user-images.githubusercontent.com/12934669/224084950-534bf7be-87da-49d5-8e28-22e5d9d60f11.png)

![image](https://user-images.githubusercontent.com/12934669/224085007-eacfedc0-277e-4dc7-8c00-566cdb12cb6d.png)
